### PR TITLE
lints: fix golangci-lints

### PR DESCRIPTION
**Pull request by cpanato**
fix golangci-lint for reference: https://github.com/google/pullsheet/runs/2009104043

/assign @tstromberg

### CODE REVIEW

File: pkg/ghcache/ghcache.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ type blob struct {
 	Issue               github.Issue
 }
 
-func PullRequestsGet(ctx context.Context, dv *diskv.Diskv, c *github.Client, t time.Time, org string, project string, num int) (*github.PullRequest, error) {
+func PullRequestsGet(ctx context.Context, dv *diskv.Diskv, c *github.Client, t time.Time, org, project string, num int) (*github.PullRequest, error) {
```
Comment (tstromberg) [inline pkg/ghcache/ghcache.go:30]: I'll be honest - I don't actually agree with this style change. I understand it's possible in Go, but I don't feel like this change makes the function signature any more readable. I think we should filter out this particular golangci-lint suggestion - do you know which checker it was?

Comment (cpanato) [reply pkg/ghcache/ghcache.go:30]: I will revert that and add the nolint rule for that

Comment (cpanato) [reply pkg/ghcache/ghcache.go:43]: done.
thanks for your review again :)

**Comment by tstromberg** [comment]
Thank you!

**Pull request merged**